app/helpers/audits_helper.rb

```diff
@@ -116,15 +121,17 @@ def audited_icon(audit)
     type   = audited_type(audit)
     symbol = case type
                when "Host"
-                 'hdd'
+                 'server'
                when "Hostgroup"
-                 'tasks'
+                 'server-group'
+               when 'Interface'
+                 'network'
                when "User"
                  'user'
                else
-                 'cog'
+                 'settings'
```

Comment:
this changes the icon for all other audits, I don't disagree with the change but is it the best icon to represent other audits?

before
![before](https://user-images.githubusercontent.com/109773/31897466-c5544700-b816-11e7-8220-fb33ffc73705.png)

after
![after](https://user-images.githubusercontent.com/109773/31897468-c697b566-b816-11e7-8c51-e2e73c75aad8.png)

Reply:
perhaps @Rohoover could suggest better generic icon or confirm this one is ok

Reply:
Basically, I changed the icons to patternfly recommendations. If there are other suggestions, I'm happy to change them.

Reply:
Would it be worth aligning these icons to the section of the application it affected/was initiated in... so anything host related would use the host icon found in the menu, Content would use the content icon. Just a thought.

Reply:
That's what it does but we have specific incons only for ~3 objects so far. This one is default if none specific icon was set. Sometimes there's not a good icon, e.g. how would we represent things like domain, foreman proxy, operating system etc.

Reply:
Well for Operating System, it's found in the Host menu, so perhaps use the Host icon until/unless we are able to find something else.